imap_processing/glows/l1/glows_l1a_data.py

```diff
@@ -0,0 +1,411 @@
+import struct
+from dataclasses import dataclass
+from typing import ClassVar
+
+from imap_processing.glows import version
+from imap_processing.glows.l0.glows_l0_data import DirectEventL0, HistogramL0
+from imap_processing.glows.utils.constants import DirectEvent, GlowsConstants, TimeTuple
+
+
+class DataEverySecond:
+    """Mapping of name to byte for dataeverysecond structure in GLOWS L1A.
+
+    This is used to generate the housekeeping info for each direct event from the
+    compressed structure in the first 40 bytes of each direct event data field.
+
+    Attributes
+    ----------
+    mapping: dict
+        Mapping of value name to the number of bytes in the compressed data
+
+    """
+
+    mapping: ClassVar[dict] = {
```

Comment:
Is it missing one Byte? In the doc string, it say 40 bytes but I counted 39. Is that expected?